index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/styles.css">
+    <title>First Project</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <p>
+                My Logo
+            </p>
+            <img src="/assets/images/breadcrumb.png" alt="">
+        </nav>
+    </header>    
+    <section class="content">
+        <h1>I’m Mariane. <br />Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <a href="#">LET’S CONNECT</a>
+        <ul class="social-media">
+            <li><a href="https://twitter.com/javiersistemas" rel="noopener" target="_blank"><img src="/assets/images/twitter.png" alt=""></a></li>
+            <li><a href="https://www.linkedin.com/in/javier-jesus-martinez-fari%C3%B1as-b71b3968/" rel="noopener" target="_blank"><img src="/assets/images/linkedin.png" alt=""></a></li>
+            <li><a href="#" target="_blank"><img src="/assets/images/m-social.png" alt=""></a></li>
+            <li><a href="https://github.com/jlcarrascof" rel="noopener" target="_blank"><img src="/assets/images/github.png" alt=""></a></li>
+            <li><a href="#" target="_blank"><img src="/assets/images/hand.png" alt=""></a></li>
+        </ul>
```

Comment:
**Good job so far 👏 👏** However it would be better to make these few adjustments to replicate the figma design as much as possible 

- [ ] Kindly replicate the ```font-weight``` and the ```font-size``` of your text from the Figma template design 👍 
- [ ] It would be better to check the links to your icons as they are not visible on my side 🙏 

**Figma design** | **Your design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/227558899-61917fc0-e32f-433c-832e-ac4deeddf177.png) | ![image](https://user-images.githubusercontent.com/102757126/227559296-f510757f-dea0-405d-80d3-f5da46b7eab4.png)

Reply:
Hello, my brother Eric @TuyishimireEric  very weird that issue about the icons but I'm working on it to fix it asap, ok? I really appreciate your time, brother :-)

Javier.-

Reply:
If you check Mr. Eric @TuyishimireEric, I used the same weight and size than in the original design of Figma but I'm using rem instead of px. 

The trouble was at the beginning of my Css files where I only put 700 as weight for Poppins font. I fixed that and the main paragraph recognized 400 as well.  🙂

The issues with my icons ... I don't know why don't appears in your browser. I changed those images by SVG files instead of the original png. I hope that runs well this time, brother. 

Waiting your feedback,

Your friend and partner,

Javier.-